config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,13 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  '_session_id',
```

Comment:
`_session_id` is what sudo sandwich is using when one runs `heroku addons:open sudo-sandwich` so I wanted to be sure it never makes it to third party Rollbar here.